spec/presenters/tree_node_builder_belongs_to_vat_spec.rb

```diff
@@ -0,0 +1,45 @@
+describe TreeNodeBuilderBelongsToVat do
+  describe '#ext_management_system_node' do
```

Comment:
Export commonly used factories into `let` blocks here, for example `ems_folder`, `ems_azure_network`